enclave/src/openfinex/client_utils.rs

```diff
@@ -74,6 +74,18 @@ impl From<u8> for Opcode {
     }
 }
 
+/// returns true if last msg, false is more msgs follow
+/// = %x0 ; more frames of this message follow
+/// %x1 ; final frame of this message
+pub fn last_frame(byte: u8) -> bool {
+    let fin_bit = (byte >> 4) & 0b1000;
+    if fin_bit == 0 {
+        false
+    } else {
+        true
+    }
+}
```

Comment:
Is this function something you were able to look up/copy from somewhere?

Reply:
Partially. I reused the part from here: https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/add-continuation_op/enclave/src/openfinex/client_utils.rs#L92

However, looking at it with a fresh mind - a bitshift by 7 would be enough. I'll update this part to  `let fin_bit = byte >> 7`